models/baseModels/SetupWizard/SetupWizard.ts

```diff
@@ -21,6 +21,21 @@ export function getCOAList() {
     { countryCode: 'ni', name: 'Nicaragua - Catalogo de Cuentas' },
     { countryCode: 'nl', name: 'Netherlands - Grootboekschema' },
     { countryCode: 'sg', name: 'Singapore - Chart of Accounts' },
+    { countryCode: 'th', name: 'Thailand - Chart of Accounts' },
+    { countryCode: 'us', name: 'United States - Chart of Accounts' },
+    { countryCode: 've', name: 'Venezuela - Plan de Cuentas' },
+    { countryCode: 'za', name: 'South Africa - Chart of Accounts' },
+    { countryCode: 'fr', name: 'France - Plan comptable' },
+    { countryCode: 'de', name: 'Germany - Kontenplan' },
+    { countryCode: 'it', name: 'Italy - Piano dei Conti' },
+    { countryCode: 'es', name: 'Spain - Plan de Cuentas' },
+    { countryCode: 'pt', name: 'Portugal - Plan de Contas' },
+    { countryCode: 'pl', name: 'Poland - Rejestr Kont' },
+    { countryCode: 'ro', name: 'Romania - Contabilitate' },
+    { countryCode: 'ru', name: 'Russia - Chart of Accounts' },
+    { countryCode: 'se', name: 'Sweden - Kontoplan' },
+    { countryCode: 'ch', name: 'Switzerland - Kontenplan' },
+    { countryCode: 'tr', name: 'Turkey - Chart of Accounts' },
```

Comment:
@mael-chouteau can you comment these out?

These Chart of Accounts aren't present ([ref](https://github.com/frappe/books/tree/master/fixtures/verified)), we can uncomment them as they're added.

Other than this everything else looks alright.